libcodechecker/store.py

```diff
@@ -323,6 +348,11 @@ def main(args):
     if isinstance(args.input, str):
         args.input = [args.input]
 
+    compile_commands = {}
+    if len(args.logfile) != 1:
+        LOG.warning("Only one log file can be processed right now!")
+        compile_commands = process_compile_db(args.logfile[0])
```

Comment:
`analyze` command exited when not 1 log file was given. Shouldn't it behave the same way?